src/includes/errors.rs

```diff
@@ -1,6 +1,14 @@
-#[derive(Clone, Copy, PartialEq, Eq)]
+#[derive(Clone, Copy, PartialEq)]
 #[cfg_attr(feature = "debug", derive(Debug))]
 pub enum CanError {
     UnknownMessageId(u32),
+    /// Signal parameter is not within the range
+    /// defined in the dbc
+    ParameterOutOfRange {
+        /// Minimum value defined in DBC
+        min: f64,
+        /// Maximum value defined in DBC
+        max: f64,
+    },
```

Comment:
Hm, I know this is me being focused on making this lightweight on embedded, but can you maybe instead just have it carry the message id? that should give the same information.

Reply:
Yes sure, well in a way you loose some information but I assume one can figure it out from there.